rules/cars-branding/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason: 
-title: Do you brand your cars?
+title: Do you nicely brand your cars?
```

Comment:
Personally I prefer `Do you brand your cars?` over of the new `Do you "nicely" brand your cars?`. I don't feel adding **nicely** adds anything to the rule and to me `Do you brand cars` reads better. 

Hi @tiagov8 , 
I've approved this, as to not block so if you think its fine feel free to merge 😄

Reply:
I added that based on the examples... which say 'bad' is a branded car but with ugly stickers...